### ISSUE TITLE
NullPointerException using COUNT birt 4.9.0

### ISSUE THREAD

**Issue by jopen-ux** [eclipse-birt/birt#1178]
report (based on Pojo Dataset) is working fine in birt 48 java8. We are  currently migrating to birt49 java17

with birt 49 we got the exception
```
Caused by: java.lang.NullPointerException: Cannot invoke "org.eclipse.birt.data.engine.api.aggregation.IAggrFunction.getParameterDefn()" because the return value of "org.eclipse.birt.data.engine.odi.IAggrInfo.getAggregation()" is null
	at org.eclipse.birt.data.engine.executor.aggregation.AggregationHelper.populateAggregations(AggregationHelper.java:100) :+1: 
```
line is :
	`aggrArgs[i] = new Object[aggrInfo.getAggregation().getParameterDefn().length];`

table :+1: 
![image](https://user-images.githubusercontent.com/53805195/211778982-baa99c0d-c78c-47ca-836e-eae88794e66c.png)

In the record, the AlaDtimeBegin is neither null, tried to separate tne runAndRenderTask in 2 steps (runTask then renderTask) ; exception  occurs during runTask... 
The main difference between our 2 environments is  that in Birt48 pom we are using org.eclipse.birt.runtime_4.8.0-20180626-4.8.0.jar and in 4.9 a list of birt and eclipse artifacts from maven repo. Could one miss, be a "bad" one ? or is it a known problem in 4.9 ?

**Comment by hvbtup** [comment]
Can this be the (basically) the same issue as #885 ?
Please test your report with BIRT 4.10 or newer.

**Comment by speckyspooky** [comment]
If you could provide a small example of your report with the POJO I can check it detailed.
I researched a little bit on your note and there are 2 locations where an error could appears if the object is null.
But I cannot reproduce the issue and I have to check the original cause of the "null"-result to understand why there becomes the object the null-value instead the object.

**Comment by claesrosell** [comment]
Closing this issue since it is not reproducible and the error was encountered in an older version of Birt. Strongly suspect that this problem was resolved with #885, since there were root cause and NPE-guard fixes in that one.
